packages/block-library/src/table-of-contents/edit.js

```diff
@@ -122,7 +122,7 @@ export default function TableOfContentsEdit( {
 									'Only including headings from the current page (if the post is paginated).'
 							  )
 							: __(
-									'Toggle to only include headings from the current page (if the post is paginated).'
+									'Only include headings from the current page (if the post is paginated).'
```

Comment:
Apologies, now with the correct context I see that my suggestion didn't make sense at all.

`Include headings from all pages (if the post is paginated).` makes more sense.

Reply:
Yeah, that’s correct. It had been wrong before our change. Nice catch!

Reply:
@swissspidy , I have made the suggested changes. Please have a look.